signup/signupController.js

```diff
@@ -37,13 +41,39 @@ define('signupController', [], function() {
 
 			SignupService.storeAppointments(data).then(function(response) {
 				if(typeof response !== 'undefined' && response && response.success){
-					$scope.successMessage = response.message;
+					document.body.scrollTop = document.documentElement.scrollTop = 0;
+					$scope.successMessage = $sce.trustAsHtml(response.message);
```

Comment:
This is cool.

Reply:
Yah if you don't trustAsHtml it actually just displays the code which is obviously not what we want, so to get around that, we have to do this.